libraries/Adapters/Microsoft.Bot.Builder.Adapters.WeChat/WeChatLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Adapters.WeChat
+{
+    public class WeChatLogger
```

Comment:
This class seems to just wrap IBotTelemetryClient, can we just use that instead and remove this class?

Reply:
Use ILogger just like BotframeworkAdapter, seems IBotTelemetryClient should only be used in Bot logic